app/skep/docker/task.py

```diff
@@ -16,7 +16,7 @@ def attrs(self):
             "id": attrs["ID"],
             "slot": attrs.get("Slot", None),
             "containerID": self.container_id(),
-            "nodeID": attrs["NodeID"],
+            "nodeID": attrs.get("NodeID", attrs.get("Node", None)),
```

Comment:
I couldn't find any docs on this @ Docker. Might want to review.  Using a get though definitely solves the "complete failure to run" though.

Reply:
Great, thank you very much. I will test this weekend.

We'll probably upgrade Docker version here at work soon as well.

Reply:
@kallisti5 Doing some work on Skep this weekend and changed a few things. I just got this error for the first time ! Merging now. : )